src/templates/zone.ejs

```diff
@@ -1,5 +1,9 @@
 <div class="overlay" data-zone="<%= zone.id %>" style="display:none">
     <div class="cards">
+        <div class="audio-player">
+            <span class="close-deck">&times;</span>
```

Comment:
Ok, gave it a better name. Want to give another quick test to make sure nothing broke.